Remove obsolete code (#146)
DataDog/dogstatsd-csharp-client: README.md

## Usage via the Statsd class

> `Statsd` is removed because it is not thread safe and not efficient. You should use instead `DogStatsdService` or `DogStatsd`:

Reviewer: ```suggestion
`Statsd` has been removed in v`6.0.0` because it is not thread safe and not efficient. Use `DogStatsdService` or `DogStatsd` instead:
```
